src/SW_Weather.c

```diff
@@ -527,6 +548,16 @@ void readAllWeather(
               weather_prefix
             );
         }
+
+        // Check to see if actual vapor pressure needs to be calculated
+        if(!SW_Weather.has_vp && SW_Weather.use_relHumidityMonthly) {
+
+            for(day = 0; day < MAX_DAYS; day++) {
+                allHist[yearIndex]->actualVaporPressure[day] =
+                                actualVaporPressure1(allHist[yearIndex]->r_humidity_daily[day],
+                                                     allHist[yearIndex]->temp_avg[day]);
+            }
+        }
     }
 }
 

```

Comment:
Nice job with adding LOCF capability to `generateMissingWeather()` for the new variables! Could you please update the section "2. Imputation by last-value-carried forward "LOCF" (`method` = 1)" of the documentation as well?